app/controllers/topical_events_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
   def show
     @classification = TopicalEvent.friendly.find(params[:id])
-    @policies = @classification.published_policies.includes(:translations, :document)
+    @policies = @classification.published_policies
```

Comment:
Isn't the plan to get rid of these topic pages entirely? I can't find anywhere in the views the variable is used, save for here, where it doesn't really serve any purpose: https://github.com/alphagov/whitehall/blob/master/app/views/topical_events/show.html.erb#L71